.doks/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: deploy_to_doks
+deploy_to_doks:
+	doctl kubernetes cluster kubeconfig save $${DOKS_CLUSTER_NAME} --set-current-context
+
+	helm repo add codacy-stable https://charts.codacy.com/stable
+	helm repo add codacy-unstable https://charts.codacy.com/unstable
+	helm repo add codacy-incubator https://charts.codacy.com/incubator
```

Comment:
Does it make sense to have these two repos here?

Reply:
yes, because in our .doks cluster we are installing stuff from incubator sometimes